### DISCUSSION TITLE
Nested group can't open the state-cards of entities

### DISCUSSION THREAD

**Discussion by ghost** [home-assistant/frontend#11197] [Other feature requests]
*[move[bot]](https://github.com/apps/move) commented on Nov 30, 2018, 8:31 PM UTC:*

_[Ingordigia](https://github.com/Ingordigia) commented on Apr 22, 2018, 1:02 PM UTC:_

**Home Assistant release with the issue:**

0.67.1

**Last working Home Assistant release (if known):**

N.A.

**Browser and Operating System:**

I think this is not a browser-dependent issue.
Tested combinations (all affected):

*   Android 6 - Chrome 65
*   Android 8 - Chrome 65
*   Windows10 - Firefox 60 (dev edition)

**Description of problem:**

When I create a nested group (shown as state-card) with entities that suppose to open a new state-card (for example input_datetime), the last child's state-card can't be opened and the entity value is shown as simple text.

My setup (simplified):
-> groupA(view) contains groupB
-> groupB contains a list of entities and groupC
-> groupC contains a list of entities (input_datetime & automation)

[![homeassistant](https://user-images.githubusercontent.com/12047767/39094768-59eb7b2a-4635-11e8-843e-d081da620108.gif)](https://user-images.githubusercontent.com/12047767/39094768-59eb7b2a-4635-11e8-843e-d081da620108.gif)

_This issue was moved by [zsarnett](https://github.com/zsarnett) from [home-assistant/home-assistant-polymer#1116](https://github.com/home-assistant/home-assistant-polymer/issues/1116)._

*This issue was moved by [iantrich](https://github.com/iantrich) from [home-assistant/ui-schema#206](https://github.com/home-assistant/ui-schema/issues/206).*

**Comment by Misiu** [comment]
@Ingordigia is this still an issue? Could you share Your card configuration?

**Comment by iantrich** [comment]
A dialog cannot currently open another dialog, so yes, still not supported

**Comment by Misiu** [comment]
The recording shows that editing date and time isn't working in groups.
Related issue: https://github.com/home-assistant/home-assistant-polymer/issues/2316

**Comment by bramkragten** [comment]
We should check if we can replace the rows with lovelace entity rows
